Header header
float32 bulletSpeed
uint8 bulletType
uint8 bulletFreq
uint8 hitPos
uint8 deltaReason